deployments/stat20/image/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# RStudio support
+jupyter-rsession-proxy==1.4
```

Comment:
We install a newer version explicitly in the dockerfile. Let's keep this or that?

Reply:
Fixed, thanks!

Reply:
@yuvipanda I had copied this over from the publichealth hub! Needs to be fixed there too.